diff -u a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml	2025-07-03 14:57:45.789238744 +0200
+++ b/pyproject.toml	2025-07-03 15:04:51.901219338 +0200
@@ -4,8 +4,7 @@
 description = "Simple Python interface for Graphviz"
 keywords = ["graph", "visualization", "dot", "render"]
 readme = "README.rst"
-license = "MIT"
-license-files = ["LICENSE.txt"]
+license = {text = "MIT"}
 dynamic = ["version"]
 requires-python = ">=3.9"
 classifiers = [
