--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,8 +8,8 @@
 name = "urllib3"
 description = "HTTP library with thread-safe connection pooling, file post, and more."
 readme = "README.md"
-license = "MIT"
-license-files = ["LICENSE.txt"]
+license = {text = "MIT"}
+license-files.globs = ["LICENSE.txt"]
 keywords = ["urllib", "httplib", "threadsafe", "filepost", "http", "https", "ssl", "pooling"]
 authors = [
   {name = "Andrey Petrov", email = "andrey.petrov@shazow.net"}
