diff --git a/bin/kevm b/bin/kevm
index 1d247187..faa99a03 100755
--- a/bin/kevm
+++ b/bin/kevm
@@ -96,9 +96,6 @@ run_kompile() {
                          if [[ "$(uname -s)" == 'Linux' ]]; then
                              ccopts+=( -lprocps )
                          elif [[ "$(uname -s)" == 'Darwin' ]]; then
-                             openssl_root="$(brew --prefix openssl)"
-                             brew_root="$(brew --prefix)"
-                             ccopts+=( -I${brew_root}/include -L/${brew_root}/lib -I${openssl_root}/include -L${openssl_root}/lib )
                              ccopts+=( -I${libcryptopp_dir}/include -L/${libcryptopp_dir}/lib                                     )
                          fi
                          for ccopt in "${ccopts[@]}"; do
