diff --git a/bin/kevm b/bin/kevm
index 3387330aa..2871710a5 100755
--- a/bin/kevm
+++ b/bin/kevm
@@ -78,9 +78,6 @@ run_kompile() {
                    kompile_opts+=( --plugin-include "${plugin_include}" --libff-dir "${libff_dir}" )
                    if [[ "$(uname -s)" == 'Darwin' ]]; then
                        kompile_opts+=( --target darwin )
-                       openssl_root="$(brew --prefix openssl)"
-                       brew_root="$(brew --prefix)"
-                       kompile_opts+=( --brew-root "${brew_root}" --openssl-root "${openssl_root}" --libcryptopp-dir "${libcryptopp_dir}" )
                    fi
                    ;;
         *)       fatal "Unknown backend for kompile: ${backend}" ;;
