diff --git a/bin/kevm b/bin/kevm
index a38c7d5ca..424f5fd3e 100755
--- a/bin/kevm
+++ b/bin/kevm
@@ -97,9 +97,6 @@ run_kompile() {
                    if [[ "$(uname -s)" == 'Linux' ]]; then
                        ccopts+=( -lprocps )
                    elif [[ "$(uname -s)" == 'Darwin' ]]; then
-                       openssl_root="$(brew --prefix openssl)"
-                       brew_root="$(brew --prefix)"
-                       ccopts+=( -I${brew_root}/include -L/${brew_root}/lib -I${openssl_root}/include -L${openssl_root}/lib )
                        ccopts+=( -I${libcryptopp_dir}/include -L/${libcryptopp_dir}/lib                                     )
                    fi
                    for ccopt in "${ccopts[@]}"; do
